### PR TITLE
[Docs] Adding code snippet tags

### PR DESCRIPTION
**Pull request by ggailey777**
This PR adds the code snippets required to import the sample code into the isolated process article in Learn.

### CODE REVIEW

**Comment by ggailey777** [comment]
@fabiocav can you have someone approve my adding snippet tags to the cancellation token samples?

**Review by satvu — APPROVED** [review]

File: samples/Net7Worker/EventHubCancellationToken.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public EventHubCancellationToken(ILoggerFactory loggerFactory)
 
         // Sample showing how to handle a cancellation token being received
         // In this example, the function invocation status will be "Cancelled"
+        <docsnippet_cancellation_token_throw>
```
Comment (kshyju) [inline samples/Net7Worker/EventHubCancellationToken.cs:20]: Don't we need to keep this behind a single line comment?

```suggestion 
        //<docsnippet_cancellation_token_throw>
```

**Review by kshyju — COMMENTED** [review]
I think the snippet tag is missing `//`

File: samples/Net7Worker/EventHubCancellationToken.cs (hunk 1 above)
Comment (ggailey777) [reply samples/Net7Worker/EventHubCancellationToken.cs:20]: 🤦‍♂️

**Comment by ggailey777** [comment]
OK, that was embarrassing. This is why I shouldn't try to make changes directly in GitHub ☹️

**Review by kshyju — APPROVED** [review]

**Review by liliankasem — APPROVED** [review]

**Comment by ggailey777** [comment]
Thanks everyone!

**Pull request merged**